### PR TITLE
Integrating Casc in the same repo and Using Casc

### PR DESCRIPTION
**Pull request by carlosrodlop**

### CODE REVIEW

File: blueprints/02-at-scale/k8s/secrets-values.yml.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) CloudBees, Inc.
 
-# Secret for Jenkins password. It is valid for all users: admin, developer, and guest.
-secJenkinsPass: "P4ssw0rd" # Required
+# Secret pass for admin_cbci_a belongs to CI_Admins group (see .docker/ldap/data.ldif)
```
Comment (kellie-freeman) [inline blueprints/02-at-scale/k8s/secrets-values.yml.example:3]:
```suggestion
# Secret password for admin_cbci_a that belongs to the CI_Admins group (refer to .docker/ldap/data.ldif)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ When filing an issue:
 1. Check existing open and recently closed issues to ensure someone else has not already reported the issue.
```
Comment (kellie-freeman) [inline CONTRIBUTING.md:15]:
```suggestion
1. Check existing open and recently closed [issues](https://github.com/cloudbees/terraform-aws-cloudbees-ci-eks-addon/issues) to ensure someone else has not already reported the issue.
```

File: blueprints/02-at-scale/casc/oc/variables/variables.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+variables:
+  - message: "Welcome to the CloudBees CI on EKS Add-on using Blueprint 02!"
```
Comment (kellie-freeman) [inline blueprints/02-at-scale/casc/oc/variables/variables.yaml:2]:
```suggestion
  - message: "Welcome to the CloudBees CI blueprint add-on: At scale!"
```

File: blueprints/02-at-scale/README.md [hunk 4]
```diff
@@ -95,9 +95,9 @@ When preparing to deploy, you must [customize the secrets file](#customize-secre
 > [!TIP]
```
[the remark this answers is not in the document]
Comment (carlosrodlop) [reply blueprints/02-at-scale/README.md:95]: Moving for the next iteration. Thanks for the feedback!

**Pull request merged**